app/models/Principal.scala

```diff
@@ -16,20 +16,31 @@
 
 package models
 
+import com.lucidchart.open.xtract.XmlReader
+import com.lucidchart.open.xtract.__
 import play.api.libs.json.Json
-import play.api.libs.json.OFormat
+import play.api.libs.json.Reads
+import play.api.libs.json.Writes
+import cats.syntax.all._
 
-final case class Principal(
-  name: String,
-  streetAndNumber: String,
-  postCode: String,
-  city: String,
-  countryCode: String,
-  eori: Option[String],
-  tir: Option[String]
-)
+final case class Principal(name: String, streetAndNumber: String, postCode: String, city: String, countryCode: String, eori: Option[String])
+//tir: Option[String] = None //TODO Investigate this??
```

Comment:
Can we put the Holder ID TIR in? Node is HITPC126 (have a look at unloading permission xsd)